tools/generate_configuration.cpp

```diff
@@ -79,7 +79,8 @@ void GenerateConfiguration(std::string const& game_epoch,
     serialization::GravityModel::Body const& body =
         solar_system.gravity_model_message(name);
     gravity_model_cfg << "  body {\n";
-    gravity_model_cfg << "    name                    = " << name << "\n";
+    gravity_model_cfg << "    name                    = "
+                      << (name == "Trappist-1" ? "Sun" : name) << "\n";
```

Comment:
Let's not hard-wire names.  Let's copy the code at lines 100-110 of `generate_kopernicus.cpp` to find the star (only if given by Keplerian elements) and let's use it here and below.